src/kite/messagescrubber.js

```diff
@@ -14,8 +19,8 @@ export default class MessageScrubber {
       kite: this.kite.getKiteInfo(),
       authentication: this.kite.options.auth,
       withArgs: params,
-      responseCallback(response = {}) {
-        const { error: rawErr, result } = response
+      responseCallback(response) {
+        const { error: rawErr, result } = response || {}
```

Comment:
> Couldn’t debug this anymore somehow this defined method can end up with
`response = null` even though it’s stated as `response = {}` in the
function definition. It might be a mistake in somewhere else, for now
making sure it’s an object in any case. Patches are welcome.

https://github.com/koding/kite.js/pull/58/commits/0fa40cb68b66b354c7dc44691c36f0942f51b4a3